easybuild/easyconfigs/c/Cython/Cython-0.22-goolf-1.4.10-Python-2.7.3.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = "PythonPackage"
+
+name = 'Cython'
+version = '0.22'
+
+homepage = 'http://pypi.python.org/pypi/Cython/'
+description = """The Cython language makes writing C extensions for the Python language as easy as Python itself.
+Cython is a source code translator based on the well-known Pyrex, but supports more cutting edge functionality and optimizations."""
+
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+
+python = 'Python'
+pythonver = '2.7.3'
+pythonshortver = '.'.join(pythonver.split('.')[0:2])
+pylibdir = 'lib/python%s/site-packages' % pythonshortver
+cythonsubdir = '-'.join(['Cython', version, 'py'+pythonshortver, 'linux-x86_64.egg'])
```

Comment:
I have to admit that I based this easyconfig on the existing ones and tried to change the minimum things (also I didn't know about this policy)

I make the changes and provide them asap.